src/components/UserInfo.js

```diff
@@ -1,20 +1,35 @@
 export class UserInfo {
-  constructor(profileNameSelector, profileDescriptionSelector) {
+  constructor(
+    profileNameSelector,
+    profileDescriptionSelector,
+    profileAvatarSelector
+  ) {
     this._profileName = document.querySelector(profileNameSelector);
     this._profileDescription = document.querySelector(
       profileDescriptionSelector
     );
+    this._profileAvatar = document.querySelector(profileAvatarSelector);
   }
 
   getUserInfo() {
     return {
       name: this._profileName.textContent,
       about: this._profileDescription.textContent,
+      avatar: this._profileAvatar.src,
     };
   }
 
-  setUserInfo({ name: profileName, about: profileDescription }) {
+  setUserInfo({
+    name: profileName,
+    about: profileDescription,
+    avatar: profileAvatar,
+  }) {
     this._profileName.textContent = profileName;
     this._profileDescription.textContent = profileDescription;
+    this._profileAvatar.src = profileAvatar;
```

Comment:
`    this._profileAvatar.src = profileAvatar;
`

Please, use `setUserAvatar` here not to duplicate the same code